### PR TITLE
Update navbar.scss

### PR DESCRIPTION
**Pull request by DoviMaj**
fix https://github.com/TheOdinProject/theodinproject/issues/1411

### CODE REVIEW

File: app/assets/stylesheets/components/navbar.scss [hunk 1]
```diff
@@ -169,6 +169,11 @@
   }
 }
 
+@media (min-width: 768px) and (max-width: 1024px)  {
+      .navbar{
```
Comment (KevinMulhern) [inline app/assets/stylesheets/components/navbar.scss:173]: The indentation seems to be a bit off, can this and its body be indented by 2 spaces just so its consistent with the rest of the file please @DoviMaj

**Review by KevinMulhern — CHANGES_REQUESTED** [review]
Looks good @DoviMaj, just a minor indentation to resolve.

File: app/assets/stylesheets/components/navbar.scss (hunk 1 above)
Comment (DoviMaj) [reply app/assets/stylesheets/components/navbar.scss]: I've done it. Is it good now?

**Comment by KevinMulhern** [comment]
Brilliant, thanks @DoviMaj

**Review by KevinMulhern — APPROVED** [review]

**Pull request merged**